src/components/RefinementList.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div :class="suit('')" v-if="state">
+    <slot
+      :items="state.items"
+      :refine="state.refine"
+      :searchable="searchable"
+      :search-for-items="state.searchForItems"
+      :toggle-show-more="toggleShowMore"
+      :is-showing-more="state.isShowingMore"
+      :createURL="state.createURL"
+      :is-from-search="state.isFromSearch"
+      :can-refine="state.canRefine"
+      :transform-Item-To-Hit="transformItemToHit"
```

Comment:
Maybe use a more meaningful name like to inform the user that is for highlighting?